<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formConfdbFinished</class>
 <widget class="QMainWindow" name="formConfdbFinished">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>185</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>365</width>
    <height>185</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>365</width>
    <height>185</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Assistente de Configuração</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>src/config_icon.png</normaloff>src/config_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#037f8c;&quot;&gt;CONFIGURAÇÃO CONCLUÍDA!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>195</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>195</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>195</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>195</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>195</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>195</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
